features_and_tools/multi-tenancy.rst

```diff
@@ -14,9 +14,10 @@ Firely Server offers *virtual* multi-tenancy. All the resources are still stored
 The basics are simple:
 
 - On every request, the tenant is read from either a specific HTTP header or a claim in the authorization token.
+- If no tenant is specified, depending whether your operation configuration requires it, the special `shared` tenant might be used internally.
```

Comment:
'might' is not very clear wording. I suggest to be very explicit. Note also that in rst you need double backticks for inline code.
```suggestion
- If no tenant is specified, and the operation being invoked does not require a tenant, the special ``shared`` tenant is used internally.
```